src/app/game-state/activity.service.ts

```diff
@@ -1783,7 +1783,7 @@ export class ActivityService {
               const grade = this.inventoryService.consume('wood');
               if (grade >= 1){ // if the wood was found
                 this.inventoryService.addItem(this.inventoryService.generateWeapon(
-                  (grade * 5) + Math.floor(Math.log2(this.characterService.characterState.attributes.woodLore.value)), 'wood'));
+                  Math.floor(Math.pow(this.characterService.characterState.attributes.metalLore.value, grade/32)), 'wood'));
```

Comment:
how come it was changed to metal lore for the wood weapon?

Reply:
crapity crap that's copypasta